lib/reading_time.ex

```diff
@@ -16,14 +13,14 @@ defmodule ReadingTime do
       1
 
   """
-  def time(string) do
+  def time(string, words_per_minute \\ 200, split_pattern \\ [" ", "\n", "\r", "\t"]) do
```

Comment:
Could we change the optional attributes to a Keyword list? That way you are not force to set a `word_per_minute_ value if you only want to override the `split_pattern` value.

Maybe something like this might work
https://riptutorial.com/elixir/example/9102/keyword-lists-as-function-parameters